packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -59,16 +81,17 @@ export default  class FileInput extends UIPlugin {
     // ___Why not use value="" on <input/> instead?
     //    Because if we use that method of clearing the input,
     //    Chrome will not trigger change if we drop the same file twice (Issue #768).
+    // @ts-expect-error yes
     event.target.value = null // eslint-disable-line no-param-reassign
   }
 
-  handleClick () {
-    this.input.click()
+  handleClick (): void {
+    this.input?.click()
   }
 
-  render () {
+  render (): ComponentChild {
     /* http://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/ */
-    const hiddenInputStyle = {
+    const hiddenInputStyle: CSSProperties = {
```

Comment:
```suggestion
    const hiddenInputStyle: JSX.CSSProperties = {
```

Reply:
I tried it but it doesn't seem to be compatible with the type:
```
Type 'CSSProperties | undefined' is not assignable to type 'string | CSSProperties | undefined'.
  Type 'CSSProperties' is not assignable to type 'string | CSSProperties | undefined'.
    Type 'Properties<string | number, string & {}>' is not assignable to type 'CSSProperties'.
      Index signature for type 'string' is missing in type 'Properties<string | number, string & {}>'.ts(2322)
jsx.d.ts(775, 3): The expected type comes from property 'style' which is declared here on type 'HTMLAttributes<HTMLInputElement> & { defaultValue?: string | undefined; }'
```

Reply:
What if you try to use `satisfies` instead?

Reply:
that works!